client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -19,96 +20,113 @@ export const useRequestColumns = () => {
-            rowData.itemStats.availableStockOnHand,
-            maxMonthsOfStock
-          ),
+    ],
+    {
+      key: 'availableStockOnHand',
+      label: 'label.stock-on-hand',
+      description: 'description.stock-on-hand',
+      align: ColumnAlign.Left,
+
+      width: 200,
+      accessor: ({ rowData }) => {
+        const { itemStats } = rowData;
+        const { availableStockOnHand, availableMonthsOfStockOnHand } =
+          itemStats;
+
+        const monthsString = availableMonthsOfStockOnHand
+          ? `(${formatNumber.round(availableMonthsOfStockOnHand, 1)} ${t(
+              'label.months',
+              {
+                count: availableMonthsOfStockOnHand,
+              }
+            )})`
+          : '';
+        return `${availableStockOnHand} ${monthsString}`;
       },
+      getSortValue: rowData => rowData.itemStats.availableStockOnHand,
+    },
+    [
+      'monthlyConsumption',
       {
-        key: 'requestedQuantity',
-        label: 'label.requested-quantity',
-        align: ColumnAlign.Right,
         width: 150,
-        getSortValue: rowData => rowData.requestedQuantity,
+        accessor: ({ rowData }) => rowData.itemStats.averageMonthlyConsumption,
+        getSortValue: rowData => rowData.itemStats.averageMonthlyConsumption,
       },
-      GenericColumnKey.Selection,
     ],
+    {
+      key: 'targetStock',
+      label: 'label.target-stock',
+      align: ColumnAlign.Right,
+      width: 150,
+      accessor: ({ rowData }) =>
+        rowData.itemStats.averageMonthlyConsumption * maxMonthsOfStock,
+      getSortValue: rowData =>
+        rowData.itemStats.averageMonthlyConsumption * maxMonthsOfStock,
+    },
+    {
+      key: 'suggestedQuantity',
+      label: 'label.forecast-quantity',
+      description: 'description.forecast-quantity',
+      align: ColumnAlign.Right,
+      width: 200,
+      accessor: ({ rowData }) =>
+        QuantityUtils.suggestedQuantity(
+          rowData.itemStats.averageMonthlyConsumption,
+          rowData.itemStats.availableStockOnHand,
+          maxMonthsOfStock
+        ),
+      getSortValue: rowData =>
+        QuantityUtils.suggestedQuantity(
+          rowData.itemStats.averageMonthlyConsumption,
+          rowData.itemStats.availableStockOnHand,
+          maxMonthsOfStock
+        ),
+    },
+    {
+      key: 'requestedQuantity',
+      label: 'label.requested-quantity',
+      align: ColumnAlign.Right,
+      width: 150,
+      getSortValue: rowData => rowData.requestedQuantity,
+    },
+  ];
+
+  if (requireSupplierAuthorisation) {
+    columnDefinitions.push({
+      key: 'approvedQuantity',
+      label: 'label.approved-quantity',
+      align: ColumnAlign.Right,
+      accessor: ({ rowData }) =>
+        rowData.linkedRequisitionLine?.approvedQuantity,
+    });
+    columnDefinitions.push({
+      key: 'approverComment',
+      label: 'label.approver-comment',
+      accessor: ({ rowData }) => rowData.linkedRequisitionLine?.comment,
```

Comment:
I think that's fine. One will be the comment popover component and the approval comment is text. should be ok, I wondered about combining into the popover and then decided it would be so little used that it isn't worth going too deep on it.